### ISSUE TITLE
Foundry V12 Compatibility

### ISSUE THREAD

**Issue by luque-gh** [Hertzila/Ironclaw2E-FoundryVTT-System#3]
Foundry recently updated to Version 12, but the current Iron Claw package is only verified for Version 11. The Iron Claw package isn't listed in the Foundry package search menu, and existing worlds using the package just straight up can't be opened.

**Comment by Tierran** [comment]
@luque-gh Should be available now 8)

**Comment by luque-gh** [comment]
I tried to update, but received this error:
"An unexpected error occurred when downloading file auto-filled: Failed to parse URL from auto-filled"

**Comment by Tierran** [comment]
Yeah, and I see why. One of the file was supposed to be auto-updated by a script when the new release was pushed (but it didn't). I'll dig a little to see why but should be an easy fix

**Comment by Tierran** [comment]
@luque-gh I found the problem and submitted a new pull request. Once @Hertzila does a new release it should work.
